### PR TITLE
Feature/change layout and requests

### PR DESCRIPTION
**Pull request by lukasz-bien-rmtcfm-com**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -17,20 +16,15 @@ const redisClient = redis.createClient(
     }
 );
 
-redisClient.get = promisify(redisClient.get);
-redisClient.set = promisify(redisClient.set);
-
 app.use(bodyParser.json());
 
 const limiter = async (req, res, next) => {
```
Comment (rafal-ksiazek-rmtcfm-com) [inline src/index.js:21]: not needed

File: src/public/script.js [hunk 2]
```diff
@@ -4,140 +4,131 @@ const limitSelect = document.querySelector('#limit-select');
...
-        if (!result || !result.status || (result.status !== 200 && result.status !== 429)) {
-            responses.unknown++;
+        if (counter) {
+            timerDiv.innerHTML = `${counter} seconds left`;
         }
-        if (result.status === 200) {
-            responses.successful++;
+
+        if (doubleRequests) {
+            for (let i = 0; i < 2; i++) {
```
Comment (rafal-ksiazek-rmtcfm-com) [inline src/public/script.js:68]: cannot you just add here some variable?

for (let i = 0; i < variable; i++)

and remove all these doubleRequests, singleRequests variables?

Comment (rafal-ksiazek-rmtcfm-com) [reply src/public/script.js:68]: eventually - just use setInterval for 500 ms; 

const requestsToSend = 20; 
const whenToSendTick = 20 / requestsToSend; // for 5 requests, send each 4 ticks, for 10 send each 2 ticks, for 20 send each 1 tick
const tick = 0; 

if (tick % whenToSendTick === 0) {
   // send request
}

File: src/index.js (hunk 1 above)
Comment (lukasz-bien-rmtcfm-com) [reply src/index.js:21]: done

File: src/public/script.js (hunk 2 above)
Comment (lukasz-bien-rmtcfm-com) [reply src/public/script.js:68]: done

File: src/public/script.js [hunk 3]
```diff
@@ -4,140 +4,133 @@ const limitSelect = document.querySelector('#limit-select');
...
 
-        successfullMessage.style.color = 'green';
-        successfullMessage.innerHTML = `${responses.successful} requests were handled properly (status 200)`;
-        resultDiv.appendChild(successfullMessage);
-    }
+            requestsSentCount++;
 
-    if (responses.limited) {
-        const limitedMessage = document.createElement('h3');
+            const currentResultsSent = document.querySelector(`#result-${resultAmount} .sent`);
```
Comment (rafal-ksiazek-rmtcfm-com) [inline src/public/script.js:74]: cannot be this moved outside interval function?

Comment (lukasz-bien-rmtcfm-com) [reply src/public/script.js:74]: it turned out that it's not needed at all, I can use previously created element and we don't need to assign classes/id to messages :)

**Pull request merged**